openstack/compute/v2/servers/requests.go

```diff
@@ -241,6 +241,12 @@ type CreateOpts struct {
 	// Tags allows a server to be tagged with single-word metadata.
 	// Requires microversion 2.52 or later.
 	Tags []string `json:"tags,omitempty"`
+
+	// (Available from 2.9.0) Hostname specifies the hostname to configure for the
```

Comment:
Nit, a small dot slipped in.
```suggestion
	// (Available from 2.90) Hostname specifies the hostname to configure for the
```

Reply:
Thank you @mandre! Fixed!